docs/nodes/nodes-library/nodes/GoogleSheets/README.md

```diff
@@ -34,39 +34,137 @@ The final workflow should look like the following image.
 
 The Start node exists by default when you create a new workflow.
 
-
+::: v-pre
 ### 2. Set node
 
-1. Click on the ***Add Value*** button and select 'Number' from the dropdown list.
-2. Enter `id` in the ***Name*** field.
-3. Click on the ***Add Value*** button and select 'String' from the dropdown list.
-4. Enter `name` in the ***Name*** field.
-5. Enter the value for the name in the ***Value*** field.
-6. Click on ***Execute Node*** to run the node.
-
-![Using the Set node to set data to be inserted by the Google Sheets node](./Set_node.png)
+The example workflow uses the Set node to generate data that we want to add to the Google Sheet. We can use other nodes, like [HTTP Request node](../../core-nodes/HTTPRequest/README.md), to get the data that we want to add to the Google Sheet.
```

Comment:
This example workflow ...

Reply:
Can you also please run this through Grammarly?

Reply:
Not sure why, but the HTTP Request node is not rendering as a link for me. Also, the word 'node' does not need to be hyperlinked.